base/applications/mspaint/mouse.cpp

```diff
@@ -17,11 +17,15 @@ POINT ToolBase::pointStack[256] = { { 0 } };
 /* FUNCTIONS ********************************************************/
 
 void
-placeSelWin()
+placeSelWin(void)
```

Comment:
Superfluous change for cpp code, isn't it?

Reply:
Reverted.